modules/misc/xdg-mime-apps.nix

```diff
@@ -75,6 +75,24 @@ in
         the list is attempted, and so on.
       '';
     };
+
+    defaultApplicationPackages = mkOption {
+      type = types.listOf types.package;
+      default = [ ];
+      example = [
+        pkgs.eog
+        pkgs.evince
+      ];
+      description = ''
+        Packages whose `.desktop` files will be used to establish default
+        mimetype associations.
+
+        These associations are appended to the associations in
+        [](#opt-xdg.mimeApps.defaultApplications). If multiple packages
+        associate with the same mime type, then the priority among them is
+        determined by their order in the list.
```

Comment:
Interestingly, I would have expected the opposite. I could default to adding all my packages in `home.packages` to this list, then if there's a conflict I want to resolve, I can set `xdg.mimaApps.defaultApplications` in Nix to overwrite it. Having the packages overwrite the explict Nix-set value sounds more difficult to manage IMO.

Feel free to argue otherwise, I'm curious what other people's perspective is on this.